src/bin/makelor.rs

```diff
@@ -365,7 +365,7 @@ fn vertex_barycentre(vertices: &[&Vertex]) -> Option<Barycentre> {
         xx += x * dE;
         yy += y * dE;
         zz += z * dE;
-        tt += t;
+        tt += t; // Time coordinate as mean of arrival time. charge not relevant.
```

Comment:
Yes, but *why* is mean arrival time better than earliest arrival time, or latest, or some weighted mean or anything else?

I have no intuition for what the correct thing to do is, which is why I left the TODO there in the first place.

Probably, it doesn't matter, because this is a 'toy' algorithm anyway. But if we're going to remove the TODO, then I'd like to record some justification for the choice we're making.

Reply:
I mean, in a true reconstructed case the most logical would be the average of the first few signals I think. Weighting according to the size of the signal doesn't seem particularly useful since you want to get as close to 'start' as possible.

Here I think the simple mean of all is the best compromise since it's a toy. I'll try and put something along those lines without writing an essay.